yb-voyager/cmd/importData.go

```diff
@@ -436,28 +436,50 @@ func checkPrimaryKey(tableName string) bool {
 	}
 	defer conn.Close(context.Background())
 
-	var table, schema string
+	var table, schema, originalTableName string
 	if len(strings.Split(tableName, ".")) == 2 {
 		schema = strings.Split(tableName, ".")[0]
 		table = strings.Split(tableName, ".")[1]
 	} else {
 		schema = target.Schema
 		table = strings.Split(tableName, ".")[0]
 	}
+	originalTableName = table
 
-	if sourceDBType == ORACLE && !utils.IsQuotedString(table) {
+
+	if utils.IsQuotedString(table) {
+		table = strings.Trim(table, `"`)
+	  } else {
 		table = strings.ToLower(table)
+	  }
+
+	checkTableSql := fmt.Sprintf(`SELECT '%s.%s'::regclass;`, schema, originalTableName)
+	log.Infof("Running query on target DB: %s", checkTableSql)
+
+	rows, err := conn.Query(context.Background(), checkTableSql)
+	if err != nil {
+		utils.ErrExit("error in querying to check table %q is present: %s", table, err)
+	}
+	
+	if(rows.Next()){
+		log.Infof("table %s is present in DB", table)
+	} else {
+		log.Infof("table %s is not present in DB", table)
+		return false
```

Comment:
Looks like you will have to exit yb-voyager. If `import data` is called for a table, then the table MUST be present on the target.
If we return `false` here, we gain nothing with this additional check--anyway the earlier code was also returning `false` for non-existent tables.

Reply:
Yes, I think in this case when table doesn't exist, we should exit with throw error like this, right? 
```
	if(rows.Next()){
		log.Infof("table %s is present in DB", table)
	} else {
		utils.ErrExit("table %q doesn't exist in target DB", table)
	}
```

Reply:
@amit-yb , I think this else case is not even required because if the query to check table is already ran fine without error then this else is not requried because empty rows can not be returned because that query selects the table name in the schema and if error came in this query we are already exiting in line 461. I will remove this else block.

Reply:
In that case, you don't need this entire `if...else` block. Just successfully running the `SELECT 'foo.bar'::regclass;` query is enough.
Make sure the output proper message when the `err.Error()` contains `"does not exist"`.